backend/src/main/java/com/rikishi/rikishi/controller/ContestantsController.java

```diff
@@ -0,0 +1,43 @@
+package com.rikishi.rikishi.controller;
+
+import com.rikishi.rikishi.model.User;
+import com.rikishi.rikishi.model.WeightClass;
+import com.rikishi.rikishi.model.entity.Contestant;
+import com.rikishi.rikishi.model.entity.Contestants;
+import com.rikishi.rikishi.provider.ResConfigProvider;
+import com.rikishi.rikishi.service.UsersService;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ContestantsController {
+    private final UsersService usersService;
+    private final ResConfigProvider resConfigProvider;
```

Comment:
Should be `ConfigProvider`, since we would like to keep some level of abstraction: what if we will want to replace `ResConfigProvider` by something else in the future? Idk, `LocalConfigProvider`? This will require a lot of changes. Instead, we can use `ConfigProvider` everywhere and just rely on Spring DI.